### DISCUSSION TITLE
Slow load when using lot's of cards

### DISCUSSION THREAD

**Discussion by jsmit99** [laravel/nova-issues#5849] [Q&A]
- Laravel Version: 10.20.0
- Nova Version: 4.27.12
- PHP Version: 8.1
- Database Driver & Version:
- Operating System and Version: Ubuntu 22.04.2
- Browser type and version: Chrome (latest)

### Description:
My main dashboard has a lot of cards (22 atm) and which exist of 19 trend cards. It seems on the first load the page is loading at around 1 - 1.25s and then it start loading the cards. But somehow it's very slow on the first request. See the timeline / waterfall.

If I try to open a single card after I clear the cache it only takes ~400ms to get the data instead of 4 - 5s. I'm using a default forge web server at Linode (dedicated 4 CPU / 8GB ram) with a very low load.

<img width="1710" alt="image" src="https://github.com/laravel/nova-issues/assets/82087876/02c0dbad-2b32-4907-9fed-2d84078d07e3">

**Comment by crynobone** [comment]
All time based metrics requires heavy database queries and if the database index doesn't match the query it is unavoidable that you see slow loading. You have to look into improving the database queries based on the required use case.

**Comment by crynobone** [comment]
https://youtu.be/HubezKbFL7E?si=rN8u2_hw05PzUFtW

**Comment by jsmit99** [comment]
The database queries it does are on a very light table with all columns indexed. All metric

<img width="1640" alt="image" src="https://github.com/laravel/nova-issues/assets/82087876/caadb719-9e54-47d6-8067-ce0680aadd10">
